examples/echo-ts/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Client, Clova, Middleware, SpeechBuilder } from "@line/clova-cek-sdk-nodejs";
+import * as express from "express";
```

Comment:
Can we change this to `import express from 'express'` ?

we can add this `"esModuleInterop": true,` to our current `tsconfig.json` under `compilerOptions`.